drivers/bluetooth/hci/Kconfig.esp32

```diff
@@ -487,12 +487,11 @@ config ESP32_BT_LE_CRYPTO_STACK_MBEDTLS
 	bool "mbedTLS crypto stack"
 	depends on ESP32_BT_LE_SECURITY_ENABLE
 	default y
-	select MBEDTLS
+	select PSA_CRYPTO_PROVIDER
 	select MBEDTLS_ECP_C
```

Comment:
Since this driver seems to depend on legacy Mbed TLS crypto APIs, shouldn't it continue to select `MBEDTLS` directly? While it will in practice always be selected through `PSA_CRYPTO_PROVIDER` since this platform doesn't have TF-M (it's not ARM), one could envision swapping to a different PSA implementation but still use legacy Mbed TLS built-in crypto.

Reply:
Oh right, I just checked that driver and it seems to me that's completely relying on legacy crypto so I really don't get why they would need `MBEDTLS_PSA_CRYPTO_C` to be enabled in the build. I will revert this change and just remove `MBEDTLS_PSA_CRYPTO_C` from the original configuration. Thanks for the tip